src/taskProvider.ts

```diff
@@ -338,8 +351,13 @@ export class TaskProvider implements vscode.TaskProvider {
   private appDeleteExec(): string {
     let exec = "";
     if (platform === "linux") {
+      let scpconfig = "";
+      const conf = vscode.workspace.getConfiguration("ledgerDevTools");
+      if (conf.get<boolean>("userScpPrivateKey") === true) {
+        scpconfig = "-e SCP_PRIVKEY=${SCP_PRIVKEY}";
+      }
       // Linux
-      exec = `docker exec -it  ${this.containerName} bash -c 'export BOLOS_SDK=$(echo ${getSelectedSDK()}) && make delete'`;
+      exec = `docker exec -it ${scpconfig} ${this.containerName} bash -c 'export BOLOS_SDK=$(echo ${getSelectedSDK()}) && make delete'`;
     } else if (platform === "darwin") {
```

Comment:
You cannot support other platforms (windows, macos) ?

Reply:
Done